backend/test.env

```diff
@@ -0,0 +1 @@
+TOKEN_SECRET=TEST123456
```

Comment:
there is a TOKEN_SECRET already in .env with a different PW ???

Reply:
I believe they only put the `test.env` but not `.env` into version control.